mlflow/metrics/genai/metric_definitions.py

```diff
@@ -0,0 +1,57 @@
+from typing import List, Optional
+
+from mlflow.exceptions import MlflowException
+from mlflow.metrics.base import EvaluationExample
+from mlflow.metrics.genai.make_genai_metric import make_genai_metric
+from mlflow.models import EvaluationMetric
+from mlflow.protos.databricks_pb2 import INTERNAL_ERROR, INVALID_PARAMETER_VALUE
+from mlflow.utils.class_utils import _get_class_from_string
+
+DEFAULT_MODEL = "openai:/gpt4"
+LATEST_VERSION = "v1"
+
+
+def correctness(
+    model: Optional[str] = DEFAULT_MODEL,
+    version: Optional[str] = LATEST_VERSION,
+    examples: Optional[List[EvaluationExample]] = [],
+) -> EvaluationMetric:
+    """
+    This function will create a genai metric used to evaluate the correctness of an LLM using the
+    model provided. Correctness will be assessed by whether answers align exactly with ground truth
+    for questions requiring precise responses, and by the similarity in meaning and description to
+    the ground truth for questions allowing varied descriptions.
+
+    An MlflowException will be raised if the specified version for this metric does not exist.
+
+    :param model: The model that will be used to evaluate this metric
+    :param version: The version of the correctness metric to use. Defaults to the latest version.
+    :param examples: Provide a list of examples to help the judge model evaluate the correctness
```

Comment:
Thoughts on this new wordings:

```
:param examples: Provide a list of examples to help the judge model evaluate the correctness. It is hight recommended to add examples so LLM can use these examples as reference to evaluate the new results.
```

Feel free to change the exact wordings. :)